src/test/java/com/coveo/spillway/SpillwayTest.java

```diff
@@ -384,4 +394,74 @@ public void triggersAreIgnoreIfTheStorageReturnsAnIncoherentResponse() {
 
     verify(callback, never()).trigger(any(LimitDefinition.class), any());
   }
+
+  @Test
+  public void canAddCapacityLimitOverride() {
+    LimitOverride override = LimitOverrideBuilder.of(JOHN).to(A_CAPACITY).per(A_DURATION).build();
+
+    Limit<User> userLimit =
+        LimitBuilder.of("perUser", User::getName)
+            .to(A_HIGHER_CAPACITY)
+            .per(A_DURATION)
+            .withLimitOverride(override)
+            .build();
+
+    Spillway<User> spillway = inMemoryFactory.enforce("testResource", userLimit);
+    assertThat(spillway.tryCall(john, A_CAPACITY + 10)).isFalse();
+  }
+
+  @Test
+  public void canAddExpirationLimitOverride() throws InterruptedException {
+    LimitOverride override =
+        LimitOverrideBuilder.of(JOHN).to(A_CAPACITY).per(A_SHORT_DURATION).build();
+
+    Limit<User> userLimit =
+        LimitBuilder.of("perUser", User::getName)
+            .to(A_CAPACITY)
+            .per(A_DURATION)
+            .withLimitOverride(override)
+            .build();
+
+    Spillway<User> spillway = inMemoryFactory.enforce("testResource", userLimit);
+    assertThat(spillway.tryCall(john, A_CAPACITY)).isTrue();
+    assertThat(spillway.tryCall(john, 1)).isFalse();
+
+    Thread.sleep(2000);
```

Comment:
Yup, as @putgeminmouth constantly reminds me, having a way to "control time" is the way to go in UTs. Makes them much more reliable.